packages/components/slider/src/slider.vue

```diff
@@ -50,6 +51,7 @@
         :vertical="vertical"
         :tooltip-class="tooltipClass"
         :placement="placement"
+        :persistent="persistent"
```

Comment:
It can be obtained through `inject`, just like `showTooltip`, instead of passing it through prop.